src/util/map.js

```diff
@@ -0,0 +1,38 @@
+import React, { Component } from "react";
+import mapboxgl from "mapbox-gl";
+
+mapboxgl.accessToken =
+"pk.eyJ1IjoiZ2l1bGlhbW1sIiwiYSI6ImNrbWRiZjM2cDI3eXoyem16MXRtZGZ3ajMifQ.KVY1mrgZdLwLzqcIsYOQrg";
```

Comment:
Probably best to store this in an environment variable, have you worked with those?

Reply:
Probably should start rotating this token, it's already leaked

Reply:
yeah that makes total sense. Passing the token as an environment variable and storing it in a .env file. That's the code:

`map.js`:
```mapboxgl.accessToken = 
`${process.env.REACT_APP_MAPBOX_KEY}`;
```

and the .env file:

```
REACT_APP_MAPBOX_KEY = ...
```